boot/freeldr/freeldr/CMakeLists.txt

```
@@ -135,3 +135,4 @@ list(APPEND FREELDR_BASE_SOURCE
 # endif()
 
 include(pcat.cmake)
+include(uefi.cmake)
```

Try whether the following works OK with cmake:
```suggestion
if(NOT ARCH STREQUAL "i386" OR NOT (SARCH STREQUAL "pc98" OR SARCH STREQUAL "xbox"))
include(uefi.cmake)
endif()
```